.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+env: CI=true
+sudo: false
+
+node_js:
```

Comment:
not sure though, we have lots of repos on travis in zalando org so this might help.